moviesurfer/cli/main.py

```diff
@@ -1,2 +1,19 @@
+import os
+import wget
+import zipfile
+
+# Function to download data
+def download_data():
+    output_dir = './moviesurfer/data'
+    url = 'http://files.grouplens.org/datasets/movielens/ml-25m.zip'
```

Comment:
I suggest to make it a constant inside the module and pass as an argument with defaults. In case the URL changes the function could still be useful. But that's up to you. This way it should work fine too.

Reply:
👍